app/data/models/customers.py

```diff
@@ -19,8 +18,8 @@ class Customer(Base):
     zip_code = sa.Column(sa.String(45), nullable=False)
     city = sa.Column(sa.String(100), nullable=False)
     country = sa.Column(sa.String(100), nullable=True)
-    orders = relationship("Order", back_populates="customer")
-    #  TODO: Make many-> many relations with customers and cars
+    orders = relationship('Order', back_populates='customer')
```

Comment:
orders = relationship('Order', back_populates='customer')
should back_populate, customers